index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio: Mobile First</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <nav class="container">
+
+    <div class="time-cell-wifi-batt">
+      <div class="time">9:41</div>
+      <div class="cell"><img src="image/cell.png" alt="cell"></div>
+      <div class="wifi"><img src="image/wifi.png" alt="wifi"></div>
+      <div class="batt"><img src="image/batt.png" alt="batt"></div>
+    </div>
+    <div class="menu"><img src="image/menu-btn.png"></div>
+    <div class="logo"><img src="image/my-logo.png" alt="My Logo"></div>
+
+    <div class="container-headline">
+      <div class="primary-text">
```

Comment:
- [ ] The project requires you to create an headline section using semantic html tags as they are the preferred standards for SEO optimization and covers and caters for every possible user of the website. visit this [link](https://www.simplilearn.com/tutorials/html-tutorial/html-semantics#:~:text=The%20semantic%20HTML%20tags%20help,offers%20a%20better%20user%20experience.) to learn more about using semantic tags.
- [ ] below is an example of how this code should be structured in your html file
```
<main>
     <section class="headline">
     </section>
</main>
```